orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/Message.kt

```diff
@@ -130,6 +133,16 @@ data class StartStage(
     this(source.getExecution().javaClass, source.getExecution().getId(), source.getExecution().getApplication(), source.getId())
 }
 
+data class ContinueParentStage(
```

Comment:
Introduced a new message triggered when a BEFORE stage with no downstream BEFORE stages completes (can also be parallel stages)